ibc/2025/0409_galaxy.md

```diff
@@ -0,0 +1,46 @@
+# IBC Eureka vulnerabity - Galaxy - 2025-04-09
+
+## What happened?
+
+Asymmetric Research discovered a critical vulnerability in the Ethereum light client we were using on the Cosmos Hub and Lombard. We proceeded with an emergency critical upgrade on the Hub, which gave us the authorization to migrate IBC Wasm light clients.
+
+## Root cause
+
+The Ethereum light client did not correctly verify the sync committee, i.e. did not verify that a pubkey is part of the committee. Additionally, an issue with pubkey verification allowed to use non-existing public keys, allowing to bypass the sync commitee signature check.
+
+The reason we needed an emergency patch on the Hub is because we expected significant volume of tokens between Ethereum and the Hub. We could not migrate the light client itself in the upgrade, which meant that we needed to be given rights to migrate the light client ourselves.
+
+Sync committee size verification was dropped because we had to support two hardforks at the same time and could not do it properly because of deadlines.
+
+## Action items
+
+- Set up security council on the Hub, similar to the one on Ethereum
```

Comment:
I think it would be good to just have a single sentence saying that these are the things we are planning to implement to prevent this in the future or something like that. And each item seem like they need a bit more context to be understandable.

Separate from that: are we tracking these anywhere?